keyboards/neito/info.json

```diff
@@ -0,0 +1,365 @@
+{
```

Comment:
Please redo this file without the rotation - it currently renders like this in the Configurator (you can preview it by pressing Ctrl+Shift+I and selecting the info.json, turning on Fast Input also helps verify the ordering matches the layout macro):

![image](https://user-images.githubusercontent.com/4781841/109741658-99bd0a00-7c21-11eb-91ce-6330840c307d.png)

Reply:
This is still not quite right, there is an offset and some rounding errors.